### ISSUE TITLE
Fix accessibility issues on header

### ISSUE THREAD

**Issue by arslanashraf7** [mitodl/mitxonline#154]
This ticket is a part of #150 EPIC.

The fixes that should be done under the banner of this ticket are:

- [x] Change `aria-label` attributes on "Sign In" and "Create Account" buttons to match the text of the buttons, instead of them both being "Login". 
- [x] Change the (duplicate) alt tag on the MIT logo from "MITx Online" to "MIT Home page" 
- [x] Make Aria ID's Unique for `div#dropdownMenuButton.col-2.dropdown-toggle` (This came after rebase on main so i added it here)

**Update: It makes sense to fo the below in another ticket/PR**
~~This should be done at the finalizing state of all the accessibility fixes~~
~~- [ ] Use the Lighthouse tool to review accessibility of all other page templates and make any necessary changes~~

**Comment by arslanashraf7** [comment]
Closed via #168
